mobidam-sst-management-backend/src/main/java/de/muenchen/mobidam/service/ZuordnungService.java

```diff
@@ -0,0 +1,43 @@
+package de.muenchen.mobidam.service;
+
+import de.muenchen.mobidam.domain.Zuordnung;
+import de.muenchen.mobidam.domain.dtos.ZuordnungCreateDTO;
+import de.muenchen.mobidam.domain.dtos.ZuordnungDTO;
+import de.muenchen.mobidam.domain.mappers.ZuordnungMapper;
+import de.muenchen.mobidam.exceptions.SavingException;
+import de.muenchen.mobidam.repository.ZuordnungRepository;
+import java.util.Optional;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@AllArgsConstructor
+@Service
+public class ZuordnungService {
+    private final ZuordnungRepository zuordnungRepository;
+    private final ZuordnungMapper zuordnungMapper;
+
+    public ZuordnungDTO create(ZuordnungCreateDTO zuordnungCreateDTO) {
+        try {
+            Zuordnung zuordnungEntity = zuordnungMapper.toEntity(zuordnungCreateDTO);
+            Zuordnung savedZuordnung = zuordnungRepository.save(zuordnungEntity);
+            return zuordnungMapper.toDTO(savedZuordnung);
+        } catch (Exception exception) {
+            throw new SavingException("Fehler beim Anlegen der Zuordnung!");
+        }
+    }
```

Comment:
```suggestion
    public ZuordnungDTO create(ZuordnungCreateDTO zuordnungCreateDTO) {
          return zuordnungMapper.toDTO(zuordnungRepository.save(zuordnungMapper.toEntity(zuordnungCreateDTO)));
    }
```

Das Exception Handling sollte in einen globalen Handler ausgelagert werden, Beispiel https://medium.com/@aedemirsen/spring-boot-global-exception-handler-842d7143cf2a

Insbesondere können dort die `ValidationException` und die `DataAccessException` (https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/dao/DataAccessException.html) behandelt werden.